blog/2025-10-22-help-us-test-opentofu-1-11-0-beta1.md

```diff
@@ -0,0 +1,245 @@
+
+### The `enabled` Meta-Argument
+
+The new `enabled` meta-argument provides a cleaner alternative to `count` and `for_each` for scenarios where you need to conditionally create a single resource or module instance. This addresses one of the most common patterns in infrastructure code: "create this resource only if a certain condition is true."
+
+#### Why `enabled`?
+
+Previously, you'd use `count = var.create_instance ? 1 : 0`, which works but has downsides:
+
+- Creates arrays even for single resources (requiring `[0]` indexing)
+- Less readable and intuitive than a simple boolean flag
+- Can be confusing for module consumers
+
+The `enabled` meta-argument solves this elegantly:
+
+```hcl
+resource "aws_instance" "web" {
+  ami           = "ami-12345"
+  instance_type = "t3.micro"
+
+  lifecycle {
+    enabled = var.create_instance  # Simple boolean condition
+  }
+}
+```
+
+#### Module Example
+
+This is especially useful in modules where you want to make resources optional:
+
+```hcl
+variable "enable_monitoring" {
+  type    = bool
+  default = true
+}
+
+resource "aws_cloudwatch_metric_alarm" "high_cpu" {
+  alarm_name          = "high-cpu-usage"
+  comparison_operator = "GreaterThanThreshold"
+  evaluation_periods  = 2
+  metric_name         = "CPUUtilization"
+
+  lifecycle {
+    enabled = var.enable_monitoring
+  }
+}
+```
+
+#### Nested in Lifecycle Block
+
+The `enabled` argument is nested inside the `lifecycle` block rather than at the resource level. This design choice prevents conflicts with existing input variables or resource arguments that might already be named `enabled`.
+
+For more information, refer to the enabled meta-argument documentation [here](/docs/main/language/meta-arguments/enabled/).
+
+## Additional Features and Improvements
+
+### Module Variable and Output Deprecation - Now Stable
+
+The ability to mark module variables and outputs as deprecated, which was experimental in OpenTofu 1.10, is now stable! Module authors can guide users away from legacy interfaces while maintaining backward compatibility.
+
+Mark variables as deprecated:
+
+```hcl
+variable "legacy_input" {
+  type       = string
+  default    = "default-value"
+  deprecated = "This variable is deprecated. Please use 'new_input' instead. This will be removed in v2.0 of this module."
+}
+```
+
+Mark outputs as deprecated:
+
+```hcl
+output "old_endpoint" {
+  value      = aws_instance.main.public_ip
+  deprecated = "This output is deprecated. Use 'instance_endpoint' instead. This will be removed in v2.0 of this module."
+}
+```
+
+Users will see clear warnings when using deprecated variables or outputs, helping them prepare for future module updates.
+
+For more information, see [marking variable as deprecated](https://opentofu.org/docs/main/language/values/variables/#marking-variable-as-deprecated) and [marking output as deprecated](https://opentofu.org/docs/main/language/values/outputs/#deprecated--marking-output-as-deprecated)
```

Comment:
The content behind both links contain notes about being deprecated. We should remove those since we remove the "experimental" label.

<img width="681" height="123" alt="Image" src="https://github.com/user-attachments/assets/ed1808d0-c494-4e19-aeda-0539cf935e29" />